chapters/hlsl.adoc

```diff
@@ -0,0 +1,164 @@
+// Copyright 2021 The Khronos Group, Inc.
+// Copyright 2021 Sascha Willems
+// SPDX-License-Identifier: CC-BY-4.0
+
+ifndef::chapters[:chapters:]
+
+[[hlsl-in-vulkan]]
+= HLSL in Vulkan
+
+Vulkan does not directly consume shaders in a human-readable text format, but instead uses link:./what_is_spirv.adoc[SPIR-V] as an intermediate representation. This opens the option to use shader languages other than e.g. GLSL, as long as they can target the Vulkan SPIR-V environment.
```

Comment:
```suggestion
Vulkan does not directly consume shaders in a human-readable text format, but instead uses xref:{chapters}what_is_spirv.adoc[SPIR-V] as an intermediate representation. This opens the option to use shader languages other than e.g. GLSL, as long as they can target the Vulkan SPIR-V environment.
```

Reply:
same for the other references to other chapters

Reply:
Updated this and the other link to internal files and replaced them with xref as suggested.